### PR TITLE
modified jb details

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #901.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 5728782ee9c565370ec74c018f05855f2196d0f4: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        8 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 1]
```diff
@@ -170,9 +170,9 @@ Changing the return value to:0x0
  22475 ms  -[JailbreakDetectionVC isJailbroken]
 ```
 
-Please note the two calls to `-[JailbreakDetectionVC isJailbroken]`, which correspond to two physical taps on the app's GUI.
+Note the two calls to `-[JailbreakDetectionVC isJailbroken]`, which correspond to two physical taps on the app's GUI.
 
-Frida is a very powerful and versatile tool. Refer to the [Frida documentation online](https://www.frida.re/docs/home/ "Frida Documentation") for more details.
+One more way to bypass Jailbreak detection mechanism that rely on file system checks is objection. Please [find the implementation here](https://github.com/sensepost/objection/blob/master/agent/src/ios/jailbreak.ts "jailbreak.ts").
```
Comment (TheDauntless) [inline Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: You removed the "please" from the earlier sentence, but added it here? :) I would just say

> You can find the implementation in the [Jailbreak.ts file on the Objection GitHub repository](https://github.com/sensepost/objection/blob/master/agent/src/ios/jailbreak.ts "jailbreak.ts")

Comment (TheDauntless) [reply Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: detection mechanism -> detection mechanisms

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 2]
```diff
@@ -130,13 +130,13 @@ false
 
 In this case we have bypassed the jailbreak detection of the application!
 
-Now, imagine that the application is closing immediately after detecting that the device is jailbroken. You don't have time to launch Cycript and replace the function implementation. Instead, you have to use CydiaSubstrate, employ a proper hooking function like `MSHookMessageEx`, and compile the tweak. There are [good sources](http://delaat.net/rp/2015-2016/p51/report.pdf "Jailbreak/Root Detection Evasion Study on iOS and Android") for how to do this; however, we will provide a potentially faster and more flexible approach.
+Now, imagine that the application is closing immediately after detecting that the device is jailbroken. You don't have time to launch Cycript and replace the function implementation. Instead, you have to use CydiaSubstrate, employ a proper hooking function like `MSHookMessageEx`, and compile the tweak. There are [good sources](http://delaat.net/rp/2015-2016/p51/report.pdf "Jailbreak/Root Detection Evasion Study on iOS and Android") for how to do this; however, we will provide a potentially faster and more flexible approach, which you should have heard by now a few times: Frida.
```
Comment (TheDauntless) [inline Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: > however, we will provide a potentially faster and more flexible approach, which you should have heard by now a few times: Frida.

Maybe to:

> however, by using Frida, we can more easily perform early instrumentation and we can build on our gathered skills from previous tests.

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ Once you start an application that has jailbreak detection enabled on a jailbrok
...
 
-Let's again look at bypassing jailbreak detection using the Damn Vulnerable iOS application as an example. After loading the binary into Hopper, you need to wait until the application is fully disassembled (look at the top bar to check the status). Then look for the "jail" string in the search box. You'll see two classes: `SFAntiPiracy` and `JailbreakDetectionVC`. You may want to decompile the functions to see what they are doing and, in particular, what they return.
+Let's look at bypassing jailbreak detection using again the Damn Vulnerable iOS application as an example. After loading the binary into Hopper, you need to wait until the application is fully disassembled (look at the top bar to check the status). Then look for the "jail" string in the search box. You'll see two classes: `SFAntiPiracy` and `JailbreakDetectionVC`. You may want to decompile the functions to see what they are doing and, in particular, what they return.
```
Comment (TheDauntless) [inline Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: I would move the "again" to the end of the sentence.

> Let's look at bypassing jailbreak detection using the Damn Vulnerable iOS application as an example again.

Comment (sushi2k) [reply Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: done

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md (hunk 1 above)
Comment (sushi2k) [reply Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: done

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md (hunk 2 above)
Comment (sushi2k) [reply Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: done

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md (hunk 3 above)
Comment (sushi2k) [reply Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: done

**Comment by OMTGreleaser** [comment]
Results for commit 373bba1aa28f970065459a2d5971849c67b9b7fd: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        8 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**